osv/ecosystems_test.py

```diff
@@ -146,12 +146,29 @@ def test_packagist(self):
     self.assertEqual(
         ecosystem.sort_key('1.0.0rc2'), ecosystem.sort_key('1.0.0.rc2'))
 
-    enumerated_versions = ecosystem.enumerate_versions("neos/neos", "3.3.0",
-                                                       "4.4.0")
-    self.assertIn("4.3.19", enumerated_versions)
-    self.assertIn("4.2.18", enumerated_versions)
-    self.assertIn("3.3.1", enumerated_versions)
-    self.assertIn("3.3.0", enumerated_versions)
+    enumerated_versions = ecosystem.enumerate_versions('neos/neos', '3.3.0',
+                                                       '4.4.0')
+    self.assertIn('4.3.19', enumerated_versions)
+    self.assertIn('4.2.18', enumerated_versions)
+    self.assertIn('3.3.1', enumerated_versions)
+    self.assertIn('3.3.0', enumerated_versions)
+
+    with open('osv/testdata/packagist_test_cases.txt') as file:
```

Comment:
nit: create a _TEST_DATA_DIR constant and make it relative to the file. also use os.path.join

something like

```python
_TEST_DATA_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'testdata'))